pkg/dmn/test-data/bulk-evaluation-test/sport_test.yml

```diff
@@ -0,0 +1,76 @@
+tests:
```

Comment:
I would rename this file to something like `sport.results.yml` the `_test` feels kinda weird because it is used by go. Even when it ends with .yml :smile: